### DISCUSSION TITLE
Import "generated directory" to custom iOS project

### DISCUSSION THREAD

**Discussion by noseeevil** [google/filament#4550] [Q&A]
Hello, I build filament 1.11.1 and I got beautiful iOS examples, hello-gltf project has generated directory that include resources for ibl and skybox, How I can import this resources to my custom project too?

My custom project is project from tutorial "CocoaPods Hello Triangle"
@bejado

**Comment by bejado** [comment]
The generated/ directories are populated through custom "Run Script" Build Phases in Xcode:
https://github.com/google/filament/blob/main/ios/samples/hello-ar/build-resources.sh
